<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DebugCore</class>
 <widget class="QWidget" name="DebugCore">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>907</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCheckBox" name="dbgcPrepointEn">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>20</y>
     <width>85</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>PrePoint</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dbgcPoint">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="dbgcCap">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>capture</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dbgcPrepoint">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>10</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelEvents">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Event</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignHCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelPoint">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>20</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Point</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="dbgcEvents">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>event</string>
    </property>
   </item>
  </widget>
  <widget class="QWidget" name="dbgcCanvas" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>880</width>
     <height>600</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
